### PR TITLE
Add tree drawing

### PR DESCRIPTION
**Pull request by jakub-drak-sbahi**
image

### CODE REVIEW

File: web/src/app/licenses/licenses.component.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="container">
+  <h1>Licenses</h1>
+
+  <ul>
+    <li>Northwoods Software: GOJS.
+      Non-comercial academic use in accordance with the terms of the license.</li>
+    <li>Material Icons. Available under Apache license version 2.0.</li>
```
Comment (BartoszChrostowski) [inline web/src/app/licenses/licenses.component.html:7]: If u arę using out the app logo please add 
< a href="https://www.vecteezy.com/free-vector/logo">Logo Vectors by Vecteezy< /a>

File: web/src/app/tree/tree-draw/tree-draw.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div id="sample">
+  <div id="myDiagramDiv" style="background-color: #F8F8F8; border: solid 1px black; width:100%; height:600px;"></div>
```
Comment (BartoszChrostowski) [inline web/src/app/tree/tree-draw/tree-draw.component.html:2]: It seems strange sizing, but idk if it works that's ok. Why height is not in %

**Review by BartoszChrostowski — APPROVED** [review]

File: web/src/app/tree/tree-draw/tree-draw.component.html (hunk 2 above)
Comment (jakub-drak-sbahi) [reply web/src/app/tree/tree-draw/tree-draw.component.html:2]: It works pretty good on different devices with different sizes, it doesn't work at all with percentage value. I found it somewhere like that not the first time.

File: web/src/app/licenses/licenses.component.html (hunk 1 above)
Comment (jakub-drak-sbahi) [reply web/src/app/licenses/licenses.component.html:7]: I am not using it yet. But I will note that for the future.

**Pull request merged**